src/CTA.WebForms2Blazor/ControlConverters/ListViewControlConverter.cs

```diff
@@ -21,7 +35,29 @@ public ListViewControlConverter() : base()
 
         public override HtmlNode Convert2Blazor(HtmlNode node)
         {
-            throw new NotImplementedException();
+            node.OwnerDocument.OptionOutputOriginalCase = true;
+
+            var itemPlaceHolderIdAttr = node.Attributes.AttributesWithName("itemplaceholderid").FirstOrDefault();
+            var itemPlaceHolder = itemPlaceHolderIdAttr?.Value ?? string.Empty;
+
+            if (!itemPlaceHolder.Equals(string.Empty))
+            {
+                IEnumerable<Attribute> layoutContextAttr = new List<Attribute>()
+                {
+                    new Attribute("Context", itemPlaceHolder)
+                    //"Context=" + itemPlaceHolder
+                };
+            
+                var success = UpdateInnerHtmlNode(node, "LayoutTemplate", newAttributes: layoutContextAttr);
+                if (success)
+                {
+                    //Could accidentally replace placeholders for other stuff with different IDs,
```

Comment:
That seems like a dangerous side effect, can we include IDs as well or were they excluded because of the complexity of adding them for now?

Reply:
It was excluded because in some cases we want to replace all instances matching name regardless of ID and other times we don't. I'll try to figure out a way to handle this

Reply:
Oh got it, we can do an optional parameter that defaults to null for ID and if it is not null then we use it.